custom/covid/tests/test_management_commands.py

```diff
@@ -110,7 +110,7 @@ def test_update_owner_ids(self):
         )
         investigators = LocationType.objects.create(
             domain=self.domain,
-            name='investigators',
+            name='Investigators',
```

Comment:
Shouldn't this now create a location with its `code` set?

Reply:
I just checked and without it, the value of `code` is still there as expected. Might be due to it being set [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/locations/models.py#L160)